docs/DeveloperGuide.md

```diff
@@ -720,7 +720,30 @@ _{Explain here how the data archiving feature will be implemented}_
 * 2a. The list is empty.
 
   Use case ends.
-    
+
+**Use case: UC08 -  List all notes**
+
+**MSS**
+
+1.  User requests to list notes.
+2.  Dictionote shows a list of notes which might or might not be empty.
+
+    Use case ends.
+
+**Use case: UC09 -  Show a specific note**
+
+**MSS**
+
+1.  User requests to show a specific note.
+2.  Dictionote shows note requested by the user in the rightmost panel.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The requested index is out of bounds. Then an index invalid exception will arise.
```

Comment:
should be 1a instead. check  https://nus-cs2103-ay2021s2.github.io/website/schedule/week7/topics.html#w7-1-requirements-use-cases

Reply:
Ok, changed